src/frontend/src/utilfunctions/commonUtils.ts

```diff
@@ -5,3 +5,7 @@ export const isInputEmpty = (text: string): boolean => {
   const trimmedText = text.trim();
   return trimmedText === '';
 };
+
+export const camelToFlat = (word: string): string => (
```

Comment:
The term for output like

Some Title String

is 'title case' so this could be camelToTitleCase

Reply:
For reference the other types are

Snake_Case, used by Python

Kebab-case, used in HTML

👍